bin/ofs/src/fuse.rs

```diff
@@ -641,7 +675,7 @@ impl PathFilesystem for Fuse {
             let metadata = op
                 .stat(e.name())
                 .await
-                .unwrap_or_else(|_| e.metadata().clone());
+                .unwrap_or_else(|_| e.metadata().clone().with_content_length(0));
```

Comment:
The same. The metadata during written should be maintained by ofs itself.

Reply:
Fix